manifests/params.pp

```diff
@@ -54,6 +54,7 @@
   $python_loglevel = 'info'
   $python_proxy = false
   $python_index = false
+  $python_use_epel = true
```

Comment:
What does this do?  You are defaulting to `undef` in init.pp not `$::puppetboard::params::python_use_epel`

Reply:
I think this just needs coming out.